src/client/brexit/summary.scss

```diff
@@ -35,7 +35,7 @@ $button-color-darker-still: darken($button-color-darker, 3%);
 	}
 
 	&__summary > p {
-		font: normal 18px/1.375 oFontsGetFontFamilyWithFallbacks(FinancierTextWeb);
+		font: normal 18px/1.375 oFontsGetFontFamilyWithFallbacks(FinancierDisplayWeb);
```

Comment:
I initially tried to update to the latest `o-typography` (Which depreciates `FinancierTextWeb`) but that was what was throwing that `null times zero` error. I then downgraded and forgot to change that line back.